lib/tasks/rotate.rake

```diff
@@ -0,0 +1,33 @@
+require 'ansi'
+
+namespace :rotate do
+
+  # benchmark: 100k updates in 00:28:35 with cost '800$8$1$'
+  desc 'email encryption key'
+  task email_encryption_key: :environment do
+    rotator = KeyRotator::EmailEncryption.new(ENV['NEW_COST'])
+    num_users = User.count
+    progress = new_progress_bar('Users', num_users)
+
+    User.find_in_batches.with_index do |users, batch|
+      User.transaction do
+        users.each do |user|
+          rotator.rotate(user)
+          progress.inc
+        end
+      end
+    end
+
+    progress.finish
+  end
+
+  def new_progress_bar(label, num)
+    progress = ::ANSI::Progressbar.new(label, num, STDOUT)
+    win_width = progress.__send__ :get_width
```

Comment:
`__send__` is super sketch IMO, is that the intentded usage of this gem? I've used https://github.com/jfelchner/ruby-progressbar with good results